sw/device/tests/power_virus_systemtest.c

```diff
@@ -230,15 +245,87 @@ static void init_peripheral_handles(void) {
+static void configure_pinmux(void) {
+  // Configure UART0 (console) and SW strapping pins.
+  pinmux_testutils_init(&pinmux);
+
+  // Configure GPIO max-power period indicator pin on IOA2
   CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa2,
                                         kTopEarlgreyPinmuxOutselGpioGpio0));
-  CHECK_DIF_OK(
-      dif_gpio_output_set_enabled(&gpio, /*pin=*/0, kDifToggleEnabled));
+  // UART1:
+  //    RX on IOA4
+  //    TX on IOA5
+  CHECK_DIF_OK(dif_pinmux_input_select(&pinmux,
+                                       kTopEarlgreyPinmuxPeripheralInUart1Rx,
+                                       kTopEarlgreyPinmuxInselIoa4));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa4,
+                                        kTopEarlgreyPinmuxOutselConstantHighZ));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa5,
+                                        kTopEarlgreyPinmuxOutselUart1Tx));
+
+  // UART2:
+  //    RX on IOB4
+  //    TX on IOB5
+  CHECK_DIF_OK(dif_pinmux_input_select(&pinmux,
+                                       kTopEarlgreyPinmuxPeripheralInUart2Rx,
+                                       kTopEarlgreyPinmuxInselIob4));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob4,
+                                        kTopEarlgreyPinmuxOutselConstantHighZ));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob5,
+                                        kTopEarlgreyPinmuxOutselUart2Tx));
+
+  // UART3:
+  //    RX on IOA0
+  //    TX on IOA1
+  CHECK_DIF_OK(dif_pinmux_input_select(&pinmux,
+                                       kTopEarlgreyPinmuxPeripheralInUart3Rx,
+                                       kTopEarlgreyPinmuxInselIoa0));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa0,
+                                        kTopEarlgreyPinmuxOutselConstantHighZ));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa1,
+                                        kTopEarlgreyPinmuxOutselUart3Tx));
+
+  // I2C0:
+  //    SCL on IOA7
+  //    SDA on IOA8
+  CHECK_DIF_OK(dif_pinmux_input_select(&pinmux,
+                                       kTopEarlgreyPinmuxPeripheralInI2c0Scl,
+                                       kTopEarlgreyPinmuxInselIoa7));
+  CHECK_DIF_OK(dif_pinmux_input_select(&pinmux,
+                                       kTopEarlgreyPinmuxPeripheralInI2c0Sda,
+                                       kTopEarlgreyPinmuxInselIoa8));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa7,
+                                        kTopEarlgreyPinmuxOutselI2c0Scl));
+  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa8,
+                                        kTopEarlgreyPinmuxOutselI2c0Sda));
+
+  // I2C1:
+  //    SCL on IOB9
+  //    SDA on IOB10
```

Comment:
For hyperdebug compatibility, swap them:

```suggestion
  //    SDA on IOB9
  //    SCL on IOB10
```

Reply:
fixed.